Tests/XMTPTests/ContactsTests.swift

```diff
@@ -81,4 +81,19 @@ class ContactsTests: XCTestCase {
 		result = await contacts.isDenied(fixtures.alice.address)
 		XCTAssertTrue(result)
 	}
+    
+    func testRefreshConsentListPagination() async throws {
+        let fixtures = await fixtures()
+        let contacts = fixtures.bobClient.contacts
+        let aliceAddress = fixtures.alice.address
+        try await contacts.deny(addresses: [aliceAddress])
+        let date = Date()
+        try await contacts.allow(addresses: [aliceAddress])
+
+        let result = try await contacts.consentList.load(pagination: Pagination(before: date, direction: .ascending))
```

Comment:
So load is a private function so external users can't call it. They call it via refreshConsentList. So I actually think you need to add pagination parameters to the refreshConsentList as well.

Reply:
I think we need to be strategic about this. We really only should accept a start date. Because we want to make sure the fetch all the latest and not just a couple in the middle since things could change. And we need it to always have the direction of ascending.

Reply:
Personal thought was that if they want to paginate in whatever way they can, but most if not all use cases would be ascending
However I can update to be safer for the SDK usage

Reply:
Agree with @nplasterer. Going in descending order is guaranteed to mangle your consent list state. Can't see why we'd offer that kind of footgun

Reply:
@neekolas @nplasterer Updated to just be after date so integrator can pass in their latest cached consent date